From 24ba78b0a9180d527408116c1533e75909c7f8c1 Mon Sep 17 00:00:00 2001
From: Josef Schlehofer <pepe.schlehofer@gmail.com>
Date: Sun, 24 Nov 2019 15:01:45 +0100
Subject: [PATCH] luajit: disable for powerpc

It doesn't compile.

See build log
https://downloads.openwrt.org/releases/faillogs-18.06/powerpc_8540/packages/luajit/

Output:
lj_arch.h:427:2: error: #error "No support for PPC/e500 anymore (use LuaJIT 2.0)"
 #error "No support for PPC/e500 anymore (use LuaJIT 2.0)"
  ^~~~~
lj_arch.h:418:2: error: #error "No support for PowerPC CPUs without double-precision FPU"
 #error "No support for PowerPC CPUs without double-precision FPU"
  ^~~~~

Signed-off-by: Josef Schlehofer <pepe.schlehofer@gmail.com>
---
 lang/luajit/Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/lang/luajit/Makefile b/lang/luajit/Makefile
index a35fcb1..12463db 100644
--- a/lang/luajit/Makefile
+++ b/lang/luajit/Makefile
@@ -26,7 +26,7 @@ define Package/luajit
  CATEGORY:=Languages
  TITLE:=LuaJIT
  URL:=http://www.luajit.org
- DEPENDS:=@(i386||x86_64||arm||armeb||powerpc||mips||mipsel||aarch64)
+ DEPENDS:=@(i386||x86_64||arm||armeb||mips||mipsel||aarch64)
 endef
 
 define Package/luajit/description
-- 
2.24.0

